docs/tutorial/part-eight/index.md

```diff
@@ -190,6 +190,28 @@ class Application extends React.Component {
 
 > 💡 The above example is from the [React Helmet docs](https://github.com/nfl/react-helmet#example). Check those out for more!
 
+If you got confused by our last example reference to `class Application extends React.Component`, dont be!
+
+To make it work Helmet in your blog, add the `Helmet` component in your `src/pages/index.js` file like:
```

Comment:
May be you have to change **"To make it work Helmet in your blog"** to **"To make it work in your blog"** or **"To make Helmet work in your blog"** based on the context. Please verify and change accordingly.

Reply:
Done. Thanks for the feedback!

Reply:
Thanks for quick action ! 👍